From 7d3ea928099d4df9c061e73b4867798161032e2c Mon Sep 17 00:00:00 2001
Message-Id: <patch-2.9-7d3ea928099-20210902T155617Z-avarab@gmail.com>
In-Reply-To: <cover-0.9-00000000000-20210902T155617Z-avarab@gmail.com>
References: <cover-0.9-00000000000-20210902T155617Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Thu, 2 Sep 2021 17:13:56 +0200
Subject: [PATCH 2/9] git-sh-setup: remove unused git_editor() function
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Remove the git_editor() function last referenced in
49eb8d39c78 (Remove contrib/examples/*, 2018-03-25).

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Documentation/git-sh-setup.txt | 5 -----
 git-sh-setup.sh                | 9 ---------
 2 files changed, 14 deletions(-)

diff --git a/Documentation/git-sh-setup.txt b/Documentation/git-sh-setup.txt
index 1ae15905492..2a28361cf66 100644
--- a/Documentation/git-sh-setup.txt
+++ b/Documentation/git-sh-setup.txt
@@ -40,11 +40,6 @@ die::
 usage::
 	die with the usage message.
 
-git_editor::
-	runs an editor of user's choice (GIT_EDITOR, core.editor, VISUAL or
-	EDITOR) on a given file, but error out if no editor is specified
-	and the terminal is dumb.
-
 is_bare_repository::
 	outputs `true` or `false` to the standard output stream
 	to indicate if the repository is a bare repository
diff --git a/git-sh-setup.sh b/git-sh-setup.sh
index ee6935ca455..cfedda79471 100644
--- a/git-sh-setup.sh
+++ b/git-sh-setup.sh
@@ -106,15 +106,6 @@ $LONG_USAGE")"
 	esac
 fi
 
-git_editor() {
-	if test -z "${GIT_EDITOR:+set}"
-	then
-		GIT_EDITOR="$(git var GIT_EDITOR)" || return $?
-	fi
-
-	eval "$GIT_EDITOR" '"$@"'
-}
-
 git_pager() {
 	if test -t 1
 	then
-- 
2.33.0.814.gb82868f05f3

